package.json

```diff
@@ -3,11 +3,14 @@
   "scripts": {
     "build": "buidler compile",
     "test": "buidler test",
+    "gas-reporter": "GAS_REPORTER=1 buidler test --network gasReporter",
+    "testnet": "buidler node",
     "lint": "yarn lint:sol && yarn lint:ts",
     "lint:sol": "solhint 'src/contracts/**/*.sol'",
     "lint:ts": "eslint --max-warnings 0 ."
   },
   "devDependencies": {
+    "@ethersproject/abstract-provider": "^5.0.5",
```

Comment:
The version of this package should be exactly the version used by `@nomiclabs/buidler`. Is there a way to enforce this constraint without having to update the version every time `@nomiclabs/buidler` is updated?